nautobot/core/templates/utilities/templatetags/tag.html

```diff
@@ -1,3 +1,3 @@
 {% load helpers %}
 
-{% if url_name %}<a href="{% url url_name %}?tags={{ tag.name }}">{% endif %}<span class="label label-default" style="color: {{ tag.color|fgcolor }}; background-color: #{{ tag.color }}">{{ tag }}</span>{% if url_name %}</a>{% endif %}
+{% if url_name %}<a href="{% url url_name %}?tags={{ tag.name }}">{% endif %}<span class="label" style="color: {{ tag.color|fgcolor }}; background-color: #{{ tag.color }}">{{ tag }}</span>{% if url_name %}</a>{% endif %}
```

Comment:
Needed because `label-default` now has a primary-blue border on it, and safe to remove as we were overriding all other `label-default` attributes anyway with the `style=` declaration.